platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/reconnect/VirtualLearnerTreeView.java

```diff
@@ -169,17 +168,11 @@ public Hash getNodeHash(final Long originalChild) {
         // Get the original record (which may be in cache or on disk) to get the hash. We should look at optimizing
         // this in the future, so we don't read the whole record if we don't have to (for example, we could use
         // the loadLeafHash method).
```

Comment:
This comment is (partially) outdated. This PR does what the comment suggests: just load a hash instead of a full record.

Reply:
Removed the comment